.gitignore

```diff
@@ -56,9 +56,6 @@ typings/
 # Yarn Integrity file
 .yarn-integrity
 
-# dotenv environment variables file
-.env
```

Comment:
Don't remove this

Reply:
How else is the local process.env config supposed to be populated?

Reply:
OK, as discussed in RC, this file is supposed to be passed around between the devs -- or a dev can copy the content from `.env.example`.

Reply:
Reverted.